### DISCUSSION TITLE
It's not automatically joining giveaways.

### DISCUSSION THREAD

**Discussion by Zorral** [Vedza/NitroSniperGo#229] [General]
I have used this sniper for 2 days and it doesn't seem to automatically join gaws, what should I do?

**Comment by Vedza** [comment]
The bot doesn't work with every giveaway bots.

**Reply by Zorral** [reply]
ummm it doesn't seem to work with any of the ones i am seen. Do you have a list of the working ones?

**Reply by Vedza** [reply]
I made it with GiveawayBot#2381, and I think it works with others bot, but I don't know which ones

**Reply by Zorral** [reply]
How do u tell if it’s joining gaws? Cause the ones I have checked, it didn’t join automatically

**Reply by Zorral** [reply]
the ones I have checked all seem to not be working, how do I know if it's working. (btw it's been running on and off for the past 3 days)
